invenio_search/cli.py

```diff
@@ -63,16 +63,26 @@ def check():
 
 @index.command()
 @click.option('--force', is_flag=True, default=False)
+@click.option(
+    '-i',
+    '--index-name',
+    'index_list',
+    help='List of indexes.',
+    multiple=True
+)
 @with_appcontext
 @es_version_check
-def init(force):
+def init(force, index_list):
```

Comment:
What do you think of adding the default value?
```suggestion
def init(force, index_list=[]):
```
or
```suggestion
def init(force, index_list=None):
```